tools/src/test/java/org/apache/kafka/tools/ToolsTestUtils.java

```diff
@@ -210,6 +212,29 @@ public static File tempPropertiesFile(Map<String, String> properties) throws IOE
         return org.apache.kafka.test.TestUtils.tempFile(sb.toString());
     }
 
+    /**
+     * Invoke `compute` until `predicate` is true or `waitTime` elapses.
+     *
+     * Return the last `compute` result and a boolean indicating whether `predicate` succeeded for that value.
+     *
+     * This method is useful in cases where `waitUntilTrue` makes it awkward to provide good error messages.
+     */
+    public static <T> Tuple2<T, Boolean> computeUntilTrue(Supplier<T> compute, long waitTime, long pause, Predicate<T> predicate) {
```

Comment:
this change can be reverted since it is unused now.

Reply:
Reverted